When built with USE=opengl, the macro GLAPI either doesn't get defined
or gets undefed in some header.  Header <GL/GLwMDrawA.h> seems to expect
it to be defined as 'extern', as defining it as '__attribute__((visibility("default")))'
leads to multiple definition errors.

--- a/RNA3D/RNA3D_GlobalHeader.hxx
+++ b/RNA3D/RNA3D_GlobalHeader.hxx
@@ -44,6 +44,7 @@
 // Undefining it here does the job:
 #undef __GLX_glx_h__
 #include <GL/glx.h>
+#define GLAPI extern
 #endif // OPEN_GL_WAY_TO_INCLUDE == 3
 // ------------------------------
 
